sdk/lib/rtl/CMakeLists.txt

```diff
@@ -8,19 +8,25 @@ add_definitions(
 if (CMAKE_C_COMPILER_ID STREQUAL "GNU" OR CMAKE_C_COMPILER_ID STREQUAL "Clang")
     # Enable this again. CORE-17637
     add_compile_options(-Wunused-result)
+
+    add_compile_options(-Wno-incompatible-pointer-types)
+    add_compile_options(-Wno-missing-braces)
 endif()
 
 list(APPEND RTL_WINE_SOURCE
     actctx.c
     timerqueue.c
-    wait.c
```

Comment:
`wait.c` is no longer?

Reply:
Yup it's now implemented by threadpool.c